src/utils/constants.ts

```diff
@@ -174,6 +174,8 @@ export const DEFAULT_CDD_ID = '0x00000000000000000000000000000000000000000000000
  */
 export const MINIMUM_SQ_VERSION = '12.2.0-alpha.2';
 
+export const SETTLEMENTS_V2_SQ_VERSION = '16.0.0-alpha.1';
```

Comment:
It is getting logged: https://github.com/PolymeshAssociation/polymesh-sdk/blob/8689f633a8b6807a4a194ff53dafde5f90056ab4/src/utils/internal.ts#L1426

Also, I think we should leave it as `12.2.0-alpha.2` for now, since this has dual version support. It should probably get a TODO to update like the version check calls.